### PR TITLE
Feature/file template

### PR DESCRIPTION
**Pull request by klaseskilson**
#### In short
- Angular directive for file displaying
- JSON template for `BruseFile`s
#### The directive `b-file-list`

This adds a angular directive that can be used when displaying files. It can be used by writing

``` html
<b-file-list files={{files}}></b-file-list>
```

or

``` html
<b-file-list></b-file-list>
```

If the attribute `files` isn't present, it will get the files automatically from the server.

### CODE REVIEW

**Comment by ronjagrosz** [comment]
nice :+1:

File: app/assets/javascripts/controllers/FileListCtrl.coffee [hunk 1]
```diff
@@ -0,0 +1,42 @@
...
+  # this function gets called from the directive
+  this.init = (element, attrs) ->
+    self.$element = element
+    $scope.name = attrs.name
+    if attrs.files
+      # if file is provided through the directive attributes, use those...
+      $scope.files = attrs.files
+    else
+      # ... otherwise collect them from server
+      FileHandler.collect().then((data) ->
```
Comment (tkomstadius) [inline app/assets/javascripts/controllers/FileListCtrl.coffee:20]: should there be some kind och catch here?

Comment (klaseskilson) [inline app/assets/javascripts/controllers/FileListCtrl.coffee:20]: There is a catch in [`app/assets/javascripts/factories/FileHandler.coffee`](https://github.com/tkomstadius/bruse/blob/feature/file-template/app/assets/javascripts/factories/FileHandler.coffee#L15-L18)

Comment (tkomstadius) [inline app/assets/javascripts/controllers/FileListCtrl.coffee:20]: okey!

**Comment by tkomstadius** [comment]
looks good.

**Pull request merged**